gradle-plugin/build.gradle

```diff
@@ -26,5 +24,5 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
     implementation "org.jetbrains.kotlin:kotlin-gradle-plugin-api"
 
-    embedded project(':kotlin-plugin')
+    runtimeOnly(project(':kotlin-plugin'))
```

Comment:
I couldn't get things to build using this `embedded` configuration. I hope `runtimeOnly` is a suitable replacement.

Reply:
`embedded` configuration is required for publishing, since I cheat by using the same artifact for Gradle + Kotlin plugin.
What kind of compilation issues did you see?